saleor/payment/gateways/stripe/plugin.py

```diff
@@ -464,12 +464,11 @@ def list_payment_sources(
     @classmethod
     def pre_save_plugin_configuration(cls, plugin_configuration: "PluginConfiguration"):
         configuration = plugin_configuration.configuration
-        flat_configuration = {item["name"]: item["value"] for item in configuration}
-
-        api_key = flat_configuration["secret_api_key"]
-        webhook_id = flat_configuration.get("webhook_endpoint_id")
-        webhook_secret = flat_configuration.get("webhook_secret_key")
+        flat_configuration = {item["name"]: item for item in configuration}
 
+        api_key = flat_configuration["secret_api_key"]["value"]
```

Comment:
Is it safe to assume this path always exists?

Reply:
The `validate_plugin_configuration` that is called before, is checking if the value is provided.